sirepo/nersc.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""Run Python processes in background
+
+:copyright: Copyright (c) 2016 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import re
+import subprocess
+from pykern import pkconfig
+from pykern.pkcollections import PKDict
+
+
+_ACCOUNT_NOT_FOUND = "no such fileset"
+
+VALID_TEST_ACCOUNT = "VALID_TEST_ACCOUNT"
+
+
+def assert_project(project):
+    if not project:
+        return ""
+    res = (
+        _test_res(project)
+        if pkconfig.channel_in_internal_test()
```

Comment:
Unit tests can "reach inside" modules. Define a function `_hpssquota(project)` that returns the returns the subprocess result. Remove the `if`. In the test, import the module and use monkeypatch like this:

```py
from sirepo import nersc
monkeypatch(nersc, "_hpssquota", lambda p: return mock_output[p])
```

`mock_output` will be a map of the different project values you want to test and the output from hpssquota modified to test two cases.